cvat-sdk/gen/templates/openapi-generator/api_client.mustache

```diff
@@ -126,8 +126,8 @@ class ApiClient(object):
         self.default_headers['User-Agent'] = value
 
     def _serialize_post_parameter(self, obj):
-        if isinstance(obj, (str, int, float, none_type, bool)):
-            return ('', json.dumps(obj), 'application/json')
+        if isinstance(obj, (str, int, float, bool)):
```

Comment:
Does the exclusion of `None` mean SDK users will obtain errors when trying to send `None` values in POST requests?

Reply:
Yes. But I don't think you can send `None` values as `multipart/form-data` anyway, so better to fail early.